azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/redux/action/ChatAction.kt

```diff
@@ -20,4 +21,6 @@ internal sealed class ChatAction : Action {
     class EndChat : ChatAction()
     class MessageReceived(val message: MessageInfoModel) : ChatAction()
     class MessageEdited(val message: MessageInfoModel) : ChatAction()
+    class ParticipantsAdded(val participants: RemoteParticipantsInfoModel) : ChatAction()
```

Comment:
Should be in ParticipantAction class

Reply:
also need reducers/state

Reply:
Added reducer and state in https://github.com/Azure/communication-ui-library-android/pull/481/commits/c807282316dba1734c8bd19cf43aae2b925b78fc